### ISSUE TITLE
about mean pixel file

### ISSUE THREAD

**Issue by 821736960** [machanic/AU_R-CNN#18]
Hi, do BP4D dataset and DISFA dataset share the same mean pixel file as you provide in  'pixel's mean value file'?

**Comment by machanic** [comment]
Yes, I use the same pixel file, which is located in https://drive.google.com/open?id=1y-yjOPuo7OcXA_bbNIZ0bfmV72mOBXON

**Comment by machanic** [comment]
It should be noted that the mean file only stores the avarage pixel value over the whole dataset, so this is not strictly limited to my file. You can compute your own mean file of DISFA dataset by using the script toolkit of https://github.com/machanic/AU_R-CNN/blob/master/dataset_toolkit/scripts/compute_mean.py
